src/test/java/hudson/plugins/jira/ConfigAsCodeTest.java

```diff
@@ -0,0 +1,29 @@
+package hudson.plugins.jira;
+
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import java.util.List;
+import java.util.Objects;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class ConfigAsCodeTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void should_support_configuration_as_code() throws Exception {
```

Comment:
why it's the only place you decided to use snake case? Oo

Reply:
Copy paste 🤭🙈